### PR TITLE
Game

### PR DESCRIPTION
**Pull request by Acusanno**
- Made an homepage, with a (small) description of the game, as well as a contributor section with everyone's picture.
Screenshot from 2023-02-20 12-27-11

- Linked the Create Custom Room pop-up to actual code in the back, and added a preview number to the range slider. For now it creates a room with only yourself in it.
Screenshot from 2023-02-20 12-29-23

### CODE REVIEW

**Review by emmanuel-lbs — APPROVED** [review]

File: front/src/components/Home/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            <div className='game-desc'>
+                <img width="30%" height="45%" min-width="500px" max-width="1200px" src={require("../../assets/pong-example.png")}/>
+                <text className='game-text'>
+                <span >H</span>ere is the pong game, you can play it by clicking on Game in the header.<br/>
+                    By default, the game ends when a player get 3 points. You can move right or left by pressing the appropriate arrow key.<br/>
+                    If you have the extra features on you can press both left and right to elevate your paddle in order to gain an advantage!<br/>
+                    To reduce the lag, the game runs both in the front and in the back, so that the game can anticipate frames if it doesn't receive them from the back.<br/>
+                    You can create a custom match, choose how many points are needed to win and invite a friend to join you.<br/><br/>
+
+                    When the game ends, you get or loose some points and you can go back to the game lobby to play again.
```
Comment (se-lee) [inline front/src/components/Home/Home.tsx:19]: loose -> lose

File: front/src/components/Home/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                        Seoyoung Lee:<br/>
+                        CEO of CSS and HTML.<br/>
+                        Responsible for the front of the chat as well as most of the front of the game.
+                    </text>
+                </div>
+                <div className='contributor'>
+                    <img className='avatar' src={require("../../assets/contributors/elabasqu.jpg")}/>
+                    <text className='text'>
+                        Emmanuel Labasque:<br/>
+                        CEO of Oauth and of bad taste.<br/>
```
Comment (se-lee) [inline front/src/components/Home/Home.tsx:39]: je ne suis pas d'accord

Comment (emmanuel-lbs) [reply front/src/components/Home/Home.tsx:39]: 🙆‍♂️

**Review by se-lee — COMMENTED** [review]
:ok_person: :ok_man:

**Pull request merged**